.github/workflows/ci.yml

```diff
@@ -19,15 +19,20 @@ jobs:
             compiler: gcc
           - os: ubuntu-22.04
             compiler: clang
+          - os: ubuntu-22.04-arm
+            compiler: gcc
+          - os: ubuntu-22.04-arm
+            compiler: clang
+
 
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
 
-      - name: Install project dependencies (Linux)
-        if: ${{ runner.os == 'Linux' }}
+      - name: Install cross compiler and QEMU if not running on ARM64
```

Comment:
The standard GH runner have everything required already [included](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md).